contrail-agent/hooks/contrail_agent_hooks.py

```diff
@@ -113,7 +135,18 @@ def install():
             # supervisord
             service_restart("supervisor-vrouter")
         install_dkms()
-
+        if config.get("agilio-vrouter"):
+            install_agilio()
+
+def install_agilio():
+    configure_virtioforwarder()
+    service("enable","virtio-forwarder")
+    configure_apparmor()
+    iface = config.get("physical-interface")
+    call("ifdown " + iface, shell=True)
```

Comment:
can you please tell me why you're using 'call' function instead of 'check_call'?